packages/angular/cli/commands/build.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "BuildCommandOptions",
+  "description": "Builds your app and places it into the output path (dist/ by default).",
+  "$longDescription": "",
+
+  "$aliases": [ "b" ],
+  "$scope": "in",
+  "$type": "architect",
+  "$impl": "./build#BuildCommand",
+
+  "type": "object",
+  "allOf": [
```

Comment:
Why not sure `properties`?

Reply:
is there a reason this is even a schema?  A majority of the actual fields are not even schema related.  An actual JSON file (with an associated schema) would seem more appropriate here.   The JSON schema elements can still be leveraged to define the option constraints and it would make the file far more intuitive.  There's also the advantage that the command schema can be used to validate all the command JSON files at build time (or in a CI check) to ensure proper formatting, required fields, etc.